devices/Ds18b20/samples/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="nanoFramework.CoreLibrary" version="1.12.0" targetFramework="netnano1.0" />
+  <package id="nanoFramework.Device.OneWire" version="1.3.4" targetFramework="netnano1.0" />
+  <package id="nanoFramework.Hardware.Esp32" version="1.3.6.7" targetFramework="netnano1.0" />
+  <package id="nanoFramework.Runtime.Events" version="1.10.0" targetFramework="netnano1.0" />
+  <package id="UnitsNet.nanoFramework.Temperature" version="4.140.0" targetFramework="netnano1.0" />
+</packages>
```

Comment:
Move this file to the base driver (`devices/DS18b20`) directory.

Reply:
@josesimoes 🤔 , the other drivers I have looked at, only have one, under the "sln"...

Reply:
But this project definitely requires a `packages.conf` for the main driver regardless!

Reply:
I'm not following your reasoning... 🤔 
A package.config is required for each nfproj file. At the same level. 
As for the packages path it's relative to the SLN file that contains all the nfproj. This is comming from NuGet, it's not nanoFramework specifc.

If there are bindings that have a single package.config that's wrong and has to be fixed ASAP.